hydradx/tests/test_basilisk_amm.py

```diff
@@ -0,0 +1,156 @@
+import copy
+import pytest
+import random
+from hypothesis import given, strategies as st, assume
+from hydradx.model.amm import basilisk_amm as bamm
+from hydradx.model.amm.amm import agent_dict
+
+asset_price_strategy = st.floats(min_value=0.01, max_value=1000)
+asset_quantity_strategy = st.floats(min_value=1000, max_value=10000000)
+fee_strategy = st.floats(min_value=0.0001, max_value=0.1, allow_nan=False, allow_infinity=False)
+trade_quantity_strategy = st.floats(min_value=-1000, max_value=1000)
+
+
+@st.composite
+def assets_config(draw) -> dict:
+    token_count = 2
+    return_dict = {
+        ''.join(random.choice('abcdefghijklmnopqrstuvwxyz') for _ in range(3)): draw(asset_quantity_strategy)
```

Comment:
That doesn't seem to be true, because it runs each path with the same random seed, and so it always chooses the same token names. So I don't believe coverage is impacted.

Reply:
I would still suggest targeting the property testing to the math, or at least have some easy way of running a set of tests that *just* target the math... I.e. not embedding randomized token names in the test strategies?

But I haven't actually seen any problems yet.